interfaces/acados_template/acados_template/acados_ocp_solver.py

```diff
@@ -156,6 +156,10 @@ def build(cls, code_export_dir, with_cython=False, cmake_builder: CMakeBuilder =
 
         with set_directory(code_export_dir):
             if os.name == 'nt':
+                # Automatically use CMakeBuilder on Windows if not provided
+                if cmake_builder is None:
+                    cmake_builder = CMakeBuilder()  # use default options
+                cmake_builder.exec(code_export_dir, verbose)
```

Comment:
Does this actually make it work?
I think there should also be a `cmake_builder` in `generate`.
The changes done in the integrator and the OCP solver should be analogous to each other.